lua/telescope/path.lua

```diff
@@ -21,29 +21,12 @@ path.make_relative = function(filepath, cwd)
   return filepath
 end
 
-path.shorten = (function()
-  if jit then
-    local ffi = require('ffi')
-    ffi.cdef [[
-    typedef unsigned char char_u;
-    char_u *shorten_dir(char_u *str);
-    ]]
-
-    return function(filepath)
-      if not filepath then
-        return filepath
-      end
-
-      local c_str = ffi.new("char[?]", #filepath + 1)
-      ffi.copy(c_str, filepath)
-      return ffi.string(ffi.C.shorten_dir(c_str))
-    end
-  else
-    return function(filepath)
-      return filepath
-    end
-  end
-end)()
+-- In most cases it is better to use `utils.path_shorten`
```

Comment:
I wouldn't update `telescope.path` anymore. Its deprecated now and we will remove it next week or so, with a bunch of other deprecated code. See `:help telescope.changelog` we will keep the `util.path_shorten` shorthand tho :)

Reply:
That's fair, I think this was a holdover from the previous attempt when I rebased. Have reverted this file now.